src/v2/Apps/Conversation/Components/BuyerGuaranteeMessage.tsx

```diff
@@ -0,0 +1,26 @@
+import { Flex, Text } from "@artsy/palette"
+import React from "react"
+
+export const BuyerGuaranteeMessage: React.FC = () => {
+  return (
+    <>
```

Comment:
Unless there's an adjacent element no need for fragments

Reply:
Ah good catch. I previously had adjacent elements but removed it and forgot to remove the fragment.